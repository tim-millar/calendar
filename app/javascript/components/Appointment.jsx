import React from 'react';
import { formatDate } from './utils';

class Appointment extends React.PureComponent {
	render () {
		return (
			<div className="card">
				<h3>{this.props.title}</h3>
				<p>{formatDate(this.props.time)}</p>
			</div>
		)
	}
}

export default Appointment
