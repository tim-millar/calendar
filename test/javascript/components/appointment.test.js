import React from 'react'
import { shallow } from 'enzyme'
import Appointment from 'Appointment'

describe('Appointment', () => {
  it('should render the title and date', () => {
    const appointment = shallow(
      <Appointment title={'Appointment'} time={'2018-03-30'} />,
    )
    expect(appointment).toMatchSnapshot()
  })
})
