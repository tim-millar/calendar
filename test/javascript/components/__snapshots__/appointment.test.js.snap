// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Appointment should render the title and date 1`] = `
<div
  className="card"
>
  <h3>
    Appointment
  </h3>
  <p>
    March 30 2018, 12:00:00 am
  </p>
</div>
`;
